src/libs/kata-sys-util/src/mount.rs

```diff
@@ -269,7 +270,12 @@ pub fn bind_mount_unchecked<S: AsRef<Path>, D: AsRef<Path>>(
     .map_err(|e| Error::BindMount(abs_src, dst.to_path_buf(), e))?;
 
     // Change into slave propagation mode.
-    mount(Some(""), dst, Some(""), MsFlags::MS_SLAVE, Some(""))
+    let propflag = match propagation {
```

Comment:
The function's comment doc gives
```text
Bind mount `src` to `dst` in slave mode, optionally in readonly mode if `readonly` is true.
```

Can you specify the need for this change? If the change is indeed needed, please change the comment doc on line 230.